requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.8.2
+Django==1.8.3
```

Comment:
I saw there was a warning in github. And I it seems the new release will effect our code here. You are right, I'll change it back to 1.8.2 for safety reasons. Can we also turn off the version feature in readme? It always gives me a feeling of errors there.😎😎😎

Baiyu Liu


Friday, July 10, 2015, 10:21 PM -0700 from Chris Schmitz  <notifications@github.com>:
>In  requirements.txt :
>> @@ -1,4 +1,4 @@
> -Django==1.8.2
> +Django==1.8.3
>Is there anything in your pull request that requires 1.8.3?  Everyone should be settled now on 1.8.2 so I am hesitant to switch unless it is needed.  Other than that everything looks good to go.
>—
>Reply to this email directly or  view it on GitHub .